### PR TITLE
RHEL6 guide can now also use a variable to change sshd MACs

### PR DESCRIPTION
**Pull request by mpreisler**
Improves UX for RHEL6 guide, unshadows for RHEL7.

This will also help us merge the rhel6 guide into linux_os in the future.

### CODE REVIEW

File: rhel6/guide/services/ssh/sshd_approved_macs.var [hunk 1]
```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'SSH Approved MACs by FIPS'
+
+description: "Specify the FIPS approved MACs (message authentication code) algorithms\n\tthat are used for data integrity protection by the SSH server."
```
Comment (jan-cerny) [inline rhel6/guide/services/ssh/sshd_approved_macs.var]: Why is the \t\n here?

**Comment by jan-cerny** [comment]
I have tested this and it works fine for me.

File: rhel6/guide/services/ssh/sshd_approved_macs.var (hunk 1 above)
Comment (mpreisler) [reply rhel6/guide/services/ssh/sshd_approved_macs.var]: Some of the descriptions use \n and \t and some are indented and use the `|-` yaml syntax. At some point we should reflow everything using `|-` however that's not in the scope of this PR. This PR just puts RHEL6 guide closer to Linux OS so it's using the Linux OS's guide as is.

File: shared/checks/oval/sshd_use_approved_macs.xml [hunk 2]
```diff
@@ -28,14 +29,40 @@
...
+  </ind:variable_object>
+
+  <ind:variable_state comment="approved macs" id="ste_sshd_use_approved_macs" version="1">
+    <ind:value operation="equals" datatype="string" var_ref="var_sshd_approved_macs" var_check="at least one" />
+  </ind:variable_state>
+
+  <ind:textfilecontent54_object id="obj_sshd_config_macs" version="1">
     <ind:filepath>/etc/ssh/sshd_config</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*(?i)MACs(?-i)[\s]+((hmac-sha1|hmac-sha2-256|hmac-sha2-512|hmac-sha1-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|hmac-sha2-512-etm@openssh\.com),?)+[\s]*(?:|(?:#.*))?$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*(?i)MACs(?-i)[\s]+([\w,-@]+)+[\s]*(?:|(?:#.*))?$</ind:pattern>
```
[the remark this answers is not in the document]
Comment (mpreisler) [reply shared/checks/oval/sshd_use_approved_macs.xml]: great question, @jan-cerny . however this is out of scope of this PR. let's do it in another PR. This PR is focused on bringing the rhel6 and linux_os guides closer together and avoid resolution errors on rhel6.

**Comment by jan-cerny** [comment]
ACK

**Pull request merged**